Examples/Scripts/Digitization/error_parameterisation.py

```diff
@@ -237,8 +237,8 @@ def run_error_parametriation(
 </head>
 <body>
 <div class="wrapper">
-    <a href="{previous}">Previous volume</a> | 
-    <a href="../index.html">Back to index</a> | 
+    <a href="{previous}">Previous volume</a> |
+    <a href="../index.html">Back to index</a> |
```

Comment:
this might be an interesting case. I think the newline will capture the space?

Reply:
I believe so, yes.